drivers/hmis/app/models/hmis/hud/processors/base.rb

```diff
@@ -290,6 +290,28 @@ def normalize_single_custom_field_value(cded, value)
       else
         raise "unexpected value \"#{value}\""
       end
+    when 'file'
+      case value
+      when String
+        # If the value is a string, it should be a blob ID for a file that's just been uploaded to ActiveStorage,
+        # but doesn't yet have a File record in our database.
+        blob = ActiveStorage::Blob.find_by(id: value)
+        file = Hmis::File.new
+        file.name = blob.filename
+        file.client = @processor.send(:client_factory)
+        file.enrollment = @processor.send(:enrollment_factory)
```

Comment:
What happens if there is no enrollment returned from the factory? Can we validate that? I don't think we want to store any files without enrollments. (`enrollment_id` is optional on `files`)